namespaces/init.go

```diff
@@ -82,6 +82,12 @@ func Init(container *libcontainer.Config, uncleanRootfs, consolePath string, pip
 		}
 	}
 
+	cloneFlags := GetNamespaceFlags(container.Namespaces, true)
```

Comment:
@LK4D4  Another version has been suggested by @xiaods, so you can choose any of them. Thanks.